src/lib/api/documents/serializer.js

```diff
@@ -1,20 +1,12 @@
-import { fromISO, toISODate } from '@api/date';
+import { toISODate } from '@api/date';
 import { recordResponseSerializer } from '@api/utils';
 import _get from 'lodash/get';
 import _set from 'lodash/set';
 
 const DocumentSerializers = {
   DATE_FIELDS: ['circulation.next_available_date'],
   responseSerializer: function(hit) {
-    const result = recordResponseSerializer(hit, function(metadata) {
```

Comment:
You can remove the entire serializer and DATE_FIELDS, the entire file, I believe...
Same for all of them below...

Reply:
just be careful on things not working after, especially we might have places where we use `document.pid` instead of `document.metadata.pid` because of these serializers

Reply:
I am not sure I am following ... As I understood in the discussion we had last week we agreed to remove the deserialization of the data when fetching it from ES/DB but we need to keep the serialization before the request, so we make sure that we send a `String` instead of `DateTime` object, since we store dates as `String` and if we enter a date in any of the forms we have, we create it as a `DateTime`. If we remove the entire file we would be sending `DateTime` objects and we would get an error. Maybe I am missing something ? 

If possible, I would like to discuss this IRL.

P.S: The `documents/serializer.js` file could be removed since we are not using it to serialize the dates (there are no dates).